keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/Exportable.kt

```diff
@@ -0,0 +1,11 @@
+package com.netflix.spinnaker.keel.api
+
+import com.netflix.spinnaker.keel.model.Moniker
+
+data class Exportable(
+  val account: String,
```

Comment:
Is `account` an EC2-specific concept? Or at least one that may not be common to all cloud providers?

Reply:
No, its lower level and should be required by all cloudProviders. If you deploy to 3 k8s clusters, each cluster should appear as a unique account.